utopia-remix/app/routes/projects.tsx

```diff
@@ -819,3 +823,74 @@ const ProjectCardActions = React.memo(({ project }: { project: ProjectWithoutCon
   )
 })
 ProjectCardActions.displayName = 'ProjectCardActions'
+
+const ActiveOperations = React.memo(() => {
+  const operations = useProjectsStore((store) => store.operations)
+
+  function getOperationVerb(op: Operation) {
+    switch (op.type) {
+      case 'delete':
+        return 'Deleting'
+      case 'destroy':
+        return 'Destroying'
+      case 'rename':
+        return 'Renaming'
+      case 'restore':
+        return 'Restoring'
+      default:
+        assertNever(op.type)
+    }
+  }
+
+  return (
+    <div
+      style={{
+        position: 'fixed',
+        bottom: 0,
+        right: 0,
+        margin: 10,
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 10,
+      }}
+    >
+      {operations
+        .sort((a, b) => -(a.startedAt - b.startedAt))
+        .map((op) => {
+          return (
```

Comment:
I think it would be nice to create a separate component for a single operation.

Reply:
done in [`db64baf` (#4970)](https://github.com/concrete-utopia/utopia/pull/4970/commits/db64baf71e96b24f0261c0feed2f1f4dfdaff7f0)